### DISCUSSION TITLE
Issue checklist not implemented right

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [FossifyOrg/General-Discussion#64] [General]
Rather than tick boxes it's text

**Comment by Aga-C** [comment]
If you write X inside `[ ]` it will create a tick.

**Reply by RustoMCSpit** [reply]
no i mean other issue templates like [auxios](https://github.com/OxygenCobalt/Auxio/issues/new?assignees=OxygenCobalt&labels=enhancement&projects=&template=feature-request.yml) allow for you to not even type for the checkbox

**Reply by Aga-C** [reply]
See: https://github.com/FossifyOrg/General-Discussion/issues/5#issuecomment-1863054816. Naveen is waiting until it's out of the beta stage.

**Reply by naveensingh** [reply]
I'll probably switch to it after I'm done with the release. It's not an organisation level thing so it has to be configured in each project one by one, I use a (dangerous) script for things like this.

@RustoMCSpit you can tick the boxes _after_ creating the issues. That way, you don't have to worry about the format.

**Comment by naveensingh** [comment]
Also, report things like this inside the issue management thread.
